ImageViewer/Source/ImageFadeInHandler.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ImageFadeInHandler.swift
+//  ImageViewer
+//
+//  Created by Kristian Angyal on 02/03/2016.
+//  Copyright © 2016 MailOnline. All rights reserved.
+//
+
+import Foundation
+
+final class ImageFadeInHandler {
+    
+    var presentedImages:[Int] = []
+    
```

Comment:
Is there a reason for it to public? I searched in the project, but it seems no one accesses this from the outside.